pkg/genericapiserver/options/etcd.go

```diff
@@ -83,4 +83,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&s.StorageConfig.Quorum, "etcd-quorum-read", s.StorageConfig.Quorum,
 		"If true, enable quorum read.")
+
+	fs.BoolVar(&s.StorageConfig.EnableCURLDebug, "enable-curl-debug", s.StorageConfig.EnableCURLDebug,
```

Comment:
Are you referring to the variable in vendor/github.com/coreos/etcd/client/curl.go? That's a vendored file from the coreos etcd client, not a file kubernetes controls. I wouldn't expect to necessarily expose every option the etcd client has via API server flags